test/v1/users.js

```diff
@@ -44,4 +44,23 @@ describe('/v1/users endpoints', () => {
           testFor(2, {id: 2, name: 'Vasanth'}, done);
       });
   });
+
+
```

Comment:
Like we maintain **v1/users/post.js** for posting in users table, why can we maintain same architecture here? like we create **tests/v1/users/post.js** that will handle the test cases for this route.  We maintain the same directory structure for the tests as the src code as in https://github.com/felipernb/algorithms.js/tree/master/src/test/